doc/whats_new/v0.21.rst

```diff
@@ -182,6 +182,12 @@ Support for Python 3.4 and below has been officially dropped.
 :mod:`sklearn.metrics`
 ......................
 
+- |Efficiency| Force ``joblib`` to use the ``'threading'`` backend in
+  :func:`metrics.pairwise.pairwise_distances` when ``n_jobs`` is greater than
```

Comment:
This doesn't really explain what changed. Maybe
```rst
- |Efficiency| Faster :func:`metrics.pairwise.pairwise_distances` with `n_jobs` > 1
  by using the `threading`, instead of the `multiprocessing`, parallel backend.
```
?

Reply:
instead of 'loky' :)

Reply:
yeah, I know, I meant process based parallelism generally...